cloud/opentracing/traces.go

```diff
@@ -0,0 +1,74 @@
+package opentracing
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/go-autorest/tracing"
```

Comment:
Are you sure about this module ? (https://github.com/Azure/go-autorest). In the repo, it will be out of support next year